src/Util_TEST.cc

```diff
@@ -1032,3 +1032,83 @@ TEST_F(UtilTest, LoadMesh)
   EXPECT_NE(nullptr, optimizedMesh);
   EXPECT_EQ(16u, optimizedMesh->SubMeshCount());
 }
+
+/////////////////////////////////////////////////
+TEST_F(UtilTest, MeshAxisAlignedBoundingBox)
+{
+  sdf::Mesh meshSdf;
+  math::AxisAlignedBox emptyBox, aab;
+  EXPECT_EQ(emptyBox, meshAxisAlignedBox(meshSdf));
+
+  meshSdf.SetUri("invalid_uri");
+  meshSdf.SetFilePath("invalid_filepath");
+  EXPECT_EQ(emptyBox, meshAxisAlignedBox(meshSdf));
+
+  meshSdf.SetUri("name://unit_box");
+  aab = meshAxisAlignedBox(meshSdf);
+  EXPECT_NE(emptyBox, aab);
+  EXPECT_EQ(aab.Size(), math::Vector3d::One);
+  EXPECT_EQ(aab.Min(), math::Vector3d(-0.5, -0.5, -0.5));
+  EXPECT_EQ(aab.Max(), math::Vector3d(0.5, 0.5, 0.5));
+  EXPECT_EQ(aab.Center(), math::Vector3d::Zero);
+
+  meshSdf.SetUri("duck.dae");
+  std::string filePath = common::joinPaths(std::string(PROJECT_SOURCE_PATH),
+    "test", "media", "duck.dae");
+  meshSdf.SetFilePath(filePath);
+  aab = meshAxisAlignedBox(meshSdf);
+  EXPECT_NE(emptyBox, aab);
+
+  // Expected values obtained from the mesh file using Blender:
+  auto minExp = math::Vector3d(-69.2985, 9.92937, -61.3282) * 1e-2;
+  auto maxExp = math::Vector3d(96.1799, 163.9700, 53.9252) * 1e-2;
+
+  EXPECT_EQ(minExp, aab.Min());
+  EXPECT_EQ(maxExp, aab.Max());
+  EXPECT_EQ((minExp + maxExp) / 2, aab.Center());
+  EXPECT_EQ(maxExp - minExp, aab.Size());
+}
+
+/////////////////////////////////////////////////
+TEST_F(UtilTest, TransformAxisAlignedBoxFrame)
+{
+  // Creates a pseudo-random vector with values between -max and max
+  std::srand(std::time(nullptr));
```

Comment:
include `<cstdlib>` and `<ctime>`

Reply:
Done in https://github.com/gazebosim/gz-sim/pull/2787/commits/469a44634d25c941a6bec80c4f5652c8ec160a9f